### PR TITLE
Feature/ Implement side bar component

### PR DESCRIPTION
**Pull request by JennyDoJD**
Result (updated): 
image

Ticket: [Implement Sidebar common component](https://trello.com/c/98RnuBrt/29-implement-sidebar-common-component)

### CODE REVIEW

**Comment by JennyDoJD** [comment]
Result (updated): 
image

File: practice/react-practice-one/src/components/Sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  return (
+    <aside className="sidebar-overlay">
+      <div className="sidebar-header">
+        <a href="javascript:void(0)" className="sidebar-logo">
+          <LogoIcon />
+        </a>
+      </div>
+      <div className="sidebar-body">
+        <ul className="nav-list">
+          <li className={`nav-item ${activeItem === 'home' ? 'active' : ''}`}>
```
Comment (ghost) [inline practice/react-practice-one/src/components/Sidebar/Sidebar.jsx]: I saw a lot of duplicate code, pls refactor

Comment (ghost) [reply practice/react-practice-one/src/components/Sidebar/Sidebar.jsx]: can we use a map?

Comment (JennyDoJD) [reply practice/react-practice-one/src/components/Sidebar/Sidebar.jsx]: Yes. I refactored using map in this case.

File: practice/react-practice-one/src/components/Sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      </div>
+      <div className="sidebar-body">
+        <ul className="nav-list">
+          {menuItems.map(({ name, icon: Icon }) => (
+            <li
+              key={name}
+              className={`nav-item ${activeItem === name ? 'active' : ''}`}
+            >
+              <a
+                href="javascript:void(0)"
```
Comment (ghost) [inline practice/react-practice-one/src/components/Sidebar/Sidebar.jsx]: Can we remove this ` href="javascript:void(0)"`

Comment (JennyDoJD) [reply practice/react-practice-one/src/components/Sidebar/Sidebar.jsx]: Yes

File: practice/react-practice-one/src/components/Sidebar/Sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    if (onClick) {
+      onClick(item);
+    }
+  };
+
+  return (
+    <aside className="sidebar-overlay">
+      <div className="sidebar-header">
+        <a href="javascript:void(0)" className="sidebar-logo">
```
Comment (ghost) [inline practice/react-practice-one/src/components/Sidebar/Sidebar.jsx]: @JennyDoJD pls help remove and double check before requesting review

Comment (JennyDoJD) [reply practice/react-practice-one/src/components/Sidebar/Sidebar.jsx]: I done

**Review by ghost — APPROVED** [review]

**Pull request merged**